python/ray/util/sgd/tests/test_torch_failure.py

```diff
@@ -107,7 +107,8 @@ def get(self):
 
         ray.get(dummy_handler.get.remote())
         ray.kill(dummy_handler)
-        time.sleep(1)
+        # make sure RESIZE_COOLDOWN times out
+        time.sleep(10)
```

Comment:
I think you can remove this and the test should still pass right?

Reply:
I think this sleep is needed. We need to make sure the RESIZE_COOLDOWN times out before the second `train`.

Reply:
This is the line we use to check `past_cooldown = (time.time() - self._last_resize) > RESIZE_COOLDOWN_S`.

But `self._last_resize` is initialized to `float("-inf")` so on the very first resize this will always be True and we shouldn't need to wait 10 seconds.